### PR TITLE
nrf: More cleanup and porting

### PR DESCRIPTION
**Pull request by arturo182**

### CODE REVIEW

File: ports/nrf/mphalport.c [hunk 1]
```diff
@@ -72,3 +72,13 @@ void mp_hal_stdout_tx_strn_cooked(const char *str, mp_uint_t len) {
 void mp_hal_stdout_tx_str(const char *str) {
     mp_hal_stdout_tx_strn(str, strlen(str));
 }
+
+void mp_hal_delay_ms(mp_uint_t ms)
+{
```
Comment (dhalbert) [inline ports/nrf/mphalport.c]: Put curly brace on previous line to make consistent with the rest of the code.

File: ports/nrf/mphalport.c [hunk 2]
```diff
@@ -72,3 +72,13 @@ void mp_hal_stdout_tx_strn_cooked(const char *str, mp_uint_t len) {
...
+    NRFX_DELAY_US(ms * 1000);
+}
+
+void mp_hal_delay_us(mp_uint_t us)
+{
```
Comment (dhalbert) [inline ports/nrf/mphalport.c]: etc..

**Comment by arturo182** [comment]
Muscle memory again, I'll try to get better at this! Thanks for the comments, I appreciate consistent code style as well, will fix it! :)

**Comment by arturo182** [comment]
@dhalbert I fixed the comments you posted, if every thing is to your liking, please merge at your convenience :)

**Pull request merged**